openwisp_radius/api/views.py

```diff
@@ -608,13 +608,18 @@ def post(self, request, *args, **kwargs):
                 radius_token = self.get_or_create_radius_token(
                     user, self.organization, renew=renew_required
                 )
+                p_token = PhoneToken.objects.filter(user=user).first()
```

Comment:
you need to ensure it's ordered correctly, there's no mention whatsoever of that.
If there are multiple tokens in the system, we must be 100% sure the last one (creation date) is taken and the code must be explicit about this.

**Also, the query should not be made at all if the user is active.**

Reply:
We are sure because it is ordered by [date created](https://github.com/openwisp/openwisp-radius/blob/master/openwisp_radius/base/models.py#L1156)